internal/pkg/config/matcher_test.go

```diff
@@ -296,6 +296,38 @@ Field: qux`,
 	}
 }
 
+func TestFieldTagMatcherUnmarshalling(t *testing.T) {
+	testCases := []struct {
+		desc, yaml        string
+		shouldErrorOnLoad bool
```

Comment:
Nit: prefer `wantErr`, the test already indicates that we're interested in "loading" (unmarshalling).

Reply:
Updated